.github/workflows/release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           node-version: lts/*
       - run: npm clean-install
       - run: npm audit signatures
-      - run: npx semantic-release
+      - run: npx semantic-release@21.0.2
```

Comment:
Should we add comment explaining that it's updated automatically by renovate and how? People might look at our release workflow for reference

Reply:
good idea. i'll get something added and link to our docs about it